app/utils/encryptable_transfer.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class EncryptableTransfer
+  def transfer(encryptable, receiver, sender_id)
```

Comment:
```suggestion
  def send(encryptable, receiver, sender_id)
```

Reply:
Why do we change now from transfer back to send?